src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -418,7 +417,6 @@ void SX127xDriver::Config(uint8_t bw, uint8_t sf, uint8_t cr, uint32_t freq, uin
 void SX127xDriver::Config(uint8_t bw, uint8_t sf, uint8_t cr, uint32_t freq, uint8_t preambleLen, uint8_t syncWord, bool InvertIQ, uint8_t _PayloadLength, uint32_t interval)
```

Comment:
If we're not going to do invertIQ then we should remove it from the function signature so we don't get confused later on!

Reply:
It's still used for the sx1280.

Reply:
Yeah, I know what you're saying but having a parameter that is ignored is a bit blah!